openmetadata-ui/src/main/resources/ui/src/components/SchemaTable/SchemaTable.test.tsx

```diff
@@ -78,6 +78,7 @@ const mockEntityTableProp = {
   columnName: '',
   hasTagEditAccess: true,
   tableConstraints: mockTableConstraints,
+  tablePartitioned: {} as TablePartition,
```

Comment:
```suggestion
  tablePartitioned: {} as Table['tablePartition'],
```

Reply:
Since it's a custom interface will stick to direct type assignment instead of infer